### PR TITLE
Javascript2 -week2

### PR DESCRIPTION
**Pull request by viedamayanti**

### CODE REVIEW

File: javascript/javascript2/week2/script.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  };
+});
+console.log(movieTag);
+
+let goodMovie = 0;
+let averageMovie = 0;
+let badMovie = 0;
+
+movieTag.forEach((movie) => {
+  if (movie.tag === 'Good') {
```
Comment (elilovna) [inline javascript/javascript2/week2/script.js:49]: looks good 👍 you also could try to use switch

File: javascript/javascript2/week2/script.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+});
+console.log(keywordMovies);
+console.log(`The total movies are ${keywordMovies.length}`);
+
+//Create an array of movies where a word in the title is duplicated. Fx "Star Wars: The Clone Wars" the word Wars is duplicated. Here are some madeup examples of movies with duplicated words in the title: "The three men and the pistol", "Chase three - The final chase"
+
+function duplicate(theTitle) {
+  let newMovie = theTitle.toLowerCase().split(' ');
+  newMovie = newMovie.map((title) => title.replace(':', ''));
+  let sameWord = [...new Set(newMovie)];
```
Comment (elilovna) [inline javascript/javascript2/week2/script.js:82]: good job for using Set!

**Review by elilovna — APPROVED** [review]
NIce work! looks quite clean!  good understanding of array methods.

File: javascript/javascript2/week2/hyf-bay/main.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+      console.log(product.name);
+    });
+  }
+  renderProduct(filteredProduct);
+});
+
+document.getElementById('select').addEventListener('change', (e) => {
+  const value = e.target.value;
+
+  if (value === 'cheap') {
```
[the remark this answers is not in the document]
Comment (viedamayanti) [reply javascript/javascript2/week2/hyf-bay/main.js:23]: I actually still confuse when is the best way to use switch 😅

**Pull request merged**